app/src/main/kotlin/com/intive/picover/parties/view/AddPartyContent.kt

```diff
@@ -85,7 +85,7 @@ fun AddPartyContent(
 			maxLines = 3,
 		)
 		Button(
-			modifier = Modifier,
+			modifier = Modifier.padding(bottom = 24.dp),
```

Comment:
What about rest of the BottomSheet's existing and upcoming one? It's not a suitable solution that the same padding needs to be applied in all of them.

Reply:
@MarekMacko This is not a generic bottomSheet but AddPartyBottomSheet, more specifically the content of this BottomSheet, which will not be used anywhere else. Of course, we can think about a more generic approach later.

Reply:
Sure, but the same padding needs to be applied to all BottomSheet's. And highly against "later", last months were a nice example. 
Of course you can ask someone else for approval 😎

Reply:
@MarekMacko good point. Challange Accepted 😎

Reply:
I think would be faster if I ask, does anyone have an idea why each padding works in ModalBottomSheet except the bottom padding? Have you ever had this issue @MarekMacko @BorysChajdas @marcinWisnia It does not work in AddPary nor in UpdateProfile. For example ModalBottomSheet(modifier.padding(25.dp)) works, ModalBottomSheet(modifier.padding(start = 25.dp, top=35.dp, end=2137)) works, ModalBottomSheet(modifier.padding(bottom = 69,dp)), no change. I google it, but I have not found anything helpful.

Reply:
Maybe there is no space for bottom padding? Try to wrap it in for example `Box` and check if padding could be applied that way

Reply:
@MarekMacko @BorysChajdas Code updated:

https://github.com/Android-Guild/Picover/assets/33199010/037eaa8c-55af-490b-82cb-5cbe4bcc3f14